dagql/idtui/frontend_pretty.go

```diff
@@ -430,9 +601,18 @@ func (fe *frontendPretty) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	case backgroundDoneMsg:
 		return fe, nil
 
+	case tea.MouseMsg:
+		switch msg.Button {
+		case tea.MouseButtonWheelDown:
+			fe.goDown()
+		case tea.MouseButtonWheelUp:
+			fe.goUp()
+		}
+		return fe, nil
+
 	case tea.KeyMsg:
 		switch msg.String() {
-		case "q", "esc", "ctrl+c":
+		case "q", "ctrl+c":
```

Comment:
Hm, something here is weird, it "feels" like CTRL+C now takes longer for the process to exit - there's a noticeable pause.

Reply:
It used to skip straight to an immediate interrupt because with the previous draining implementation it was too easy to end up in a scenario where you're stuck waiting on a "doomed" span (i.e. parent `kill -9`ed).

Now that that's fixed, I changed interrupt to not interrupt immediately, which gives telemetry a chance to drain, but takes a second or two. I can look into why. Maybe we have a 1 second sleep retry somewhere.